cms/djangoapps/contentstore/tests/test_contentstore.py

```diff
@@ -1473,33 +1472,6 @@ def test_item_factory(self):
         item = ItemFactory.create(parent_location=course.location)
         self.assertIsInstance(item, SequenceDescriptor)
 
-    def test_course_index_view_with_course(self):
-        """Test viewing the index page with an existing course"""
-        CourseFactory.create(display_name='Robot Super Educational Course')
-        resp = self.client.get_html('/home/')
-        self.assertContains(
-            resp,
-            '<h3 class="course-title">Robot Super Educational Course</h3>',
-            status_code=200,
-            html=True
-        )
-
-    def test_course_index_view_xss(self):
```

Comment:
Since the lists are now client-side rendered, these tests need to be removed. I tested with XSS injections and had no problems with the React code (and note that we already have bok choy test coverage for XSS for the dashboard page).